src/classes/OPP_OpportunityNaming_BATCH.cls

```diff
@@ -35,56 +33,38 @@
 * @description Batch class creates names all Opportunities per the naming spec. Batch job chunks Opportunities
 * in groups of 10,000,000 records ordered by Id and chains itself to process additional records to avoid query limits
  */
-public class OPP_OpportunityNaming_BATCH implements Database.Batchable<sObject>, Schedulable, Database.Stateful {
-    /** @description The query for the batch process to run on.*/
-    private String query;
-
-    @TestVisible
-    private static Integer defaultQueryLimit = 10000000;
+public class OPP_OpportunityNaming_BATCH extends UTIL_AbstractCommon_BATCH implements Schedulable {
+    /**
+     * @description # of records in each batch
+     */
+    private static final Integer BATCH_SIZE = 200;
 
-    @TestVisible
-    private Id lastOppIdProcessed;
+    /**
+     * @description Chunk size to use when running in LDV Mode
+     */
+    private static final Integer CHUNK_SIZE = 10000000;
 
-    /** @description The batch process constructor; creates opportunity query for all opportunities.*/
+    /** @description Default No argument constructor */
     public OPP_OpportunityNaming_BATCH() {
-        query = 'SELECT Id, Name FROM Opportunity ORDER BY Id LIMIT ' + defaultQueryLimit;
-    }
-
-    /** @description Constructor that accepts Id offset*/
-    public OPP_OpportunityNaming_BATCH(Id idToOffset) {
-        query = buildOffsetQuery(idToOffset, defaultQueryLimit);
-    }
-
-    /** @description Builds query string with given Id to offset and query limit*/
-    private String buildOffsetQuery(Id idToOffset, Integer queryLimit) {
-        return String.format(
-            'SELECT Id, Name FROM Opportunity WHERE Id > \'\'{0}\'\' ORDER BY Id LIMIT {1}',
-            new List<String>{ idToOffset, String.valueOf(queryLimit) }
-        );
+        super();
     }
 
-    /** @description Batch process start method.*/
-    public Database.QueryLocator start(Database.BatchableContext BC) {
-        return Database.getQueryLocator(query);
+    /** @description Constructor to allow specifying batch size.*/
+    public OPP_OpportunityNaming_BATCH(Integer batchSize) {
```

Comment:
@jbrandolinisf As Discussed
Follow-up for later: Refactor the overloaded constructors into a single constructor and use set___() methods to pass additional values.

Reply:
@force2b - What else do I need to do to finish this up?  Rename the test methods?  Anything else from the live review I need to do?  Rename the class?  I just want to make sure I get it in one more pass.

Reply:
Yes, we talked about renaming the test methods to reflect what was being tested. For example, "shouldProcessZeroRecords". The class rename was another, as well as [eventually] refactoring the constructors and methods simplify the implementation. Removing unused methods is optional, and though it would reduce the complexity of the class, it would require removing the related test methods as well.